Bottom: dcdc8abbd11aa4b9839d864316e832ddf3e87bf5
Top:    6735c5a3dc30a0ec4d0b33f382a89fda4dff498a
Author: Christopher Voltz <cjunk@voltz.ws>
Date:   2017-05-01 21:58:45 -0500

Add CHANGELOG.md

---

diff --git a/CHANGELOG.md b/CHANGELOG.md
new file mode 100644
index 0000000..d727d01
--- /dev/null
+++ b/CHANGELOG.md
@@ -0,0 +1,9 @@
+# Change Log
+
+## 2017-05-01
+- Added Rakefile to automate tasks used in creating the patchset.
+- Updated to 2017-02-09 upstream version of tablecalc.
+
+## 2016-04-29
+- Added tag, tag_row, and tag_col keywords which allow one to tag a cell in a
+  table and later get the row and column of the tagged cell.
